### DISCUSSION TITLE
Non-weapon artifacts

### DISCUSSION THREAD

**Discussion by rwisaak** [fvtt-fria-ligan/forbidden-lands-foundry-vtt#27] [Ideas]
As the 'artifact' item type has been removed and integrated into the 'weapon' item type, we no longer have a good place for non-weapon artifact types. _Barkhyde_, for example, is a helmet (kind of). It would fit best into the armor category, mechanically, but there's currently no way to properly add its effects, drawbacks, etc. there. Entered as an artifact weapon, it gets placed into the 'weapons' slot on a character sheet, and of course lacks a way to track armor rating there as well.

Is there currently a plan to revise the armor item sheet to allow for artifact types? There are also a fair number of artifact items that don't really fall well into either category and should maybe considered more as artifact 'gear' (_Clay's Rosary_, for example).

**Comment by aMediocreDad** [comment]
Hello @rwisaak!
The short answer is, yes. 

This is high on my priority list as the system badly needs better support for Artifacts. When will it arrive? I am not sure, but I plan to start working on it this week. Hopefully with the groundwork laid by @giant-teapot this shouldn't present too much of a challenge.
